usaspending_api/search/v2/views/spending_over_time.py

```diff
@@ -331,16 +458,32 @@ def post(self, request: Request) -> Response:
                 date_range_type=order_by_cols[-1],
                 columns={"aggregated_amount": "aggregated_amount"},
             )
-        else:
-            results = self.query_elasticsearch_for_prime_awards(time_periods)
+            for result in results:
```

Comment:
Any particular reason to add this to each response as `None` and not the other Award type fields? The API contract claims that all of these fields are required.

It does also claim the obligation fields are required even though they are not present, but I won't fault you on that because it is outside of the scope of this work.

Reply:
Updated the Subaward result object to match what is defined in the API contract.